buildSrc/src/main/java/com/uber/okbuck/core/util/FileUtil.java

```diff
@@ -156,4 +157,27 @@ public static void persistMapToJsonFile(HashMap<String, String> map, File file)
     writer.flush();
     writer.close();
   }
+
+  public static void deleteQuitelyAndCreate(File dir, boolean deleteDir, String fileName) {
+
+    // Clean the dir, if it exists, based on deleteDir config
+    if (dir.exists()) {
+      if (deleteDir) {
+        try {
+          FileUtils.deleteDirectory(dir);
+        } catch (IOException e) {
+          throw new IllegalStateException("Could not delete dependency directory: " + dir, e);
+        }
+      } else {
+        Collection<File> files = FileUtils.listFiles(dir, new NameFileFilter(fileName), null);
+        for (File file : files) {
+          FileUtils.deleteQuietly(file);
```

Comment:
would we be able to delete in parallel using parallelstream?

Reply:
ah, good point.